script/clippy

```diff
@@ -5,5 +5,5 @@ set -euxo pipefail
 # clippy.toml is not currently supporting specifying allowed lints
 # so specify those here, and disable the rest until Zed's workspace
 # will have more fixes & suppression for the standard lint set
-cargo clippy --release --workspace --all-features --all-targets -- -A clippy::all -D clippy::dbg_macro -D clippy::todo
```

Comment:
I would really love to keep the TODO check if you don't mind, and maybe replace that Linux one with a `unimplemented!("TODO ...")` thing?

Reply:
Will do :)